src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -168,15 +168,18 @@ const useClickTrackerHandler = (props = {}) => {
   );
 };
 
-export const useConstructLiteSiteATIEventTrackUrl = (props = {}) => {
-  const atiTrackingParams = extractTrackingProps(props);
+export const useConstructLiteSiteATIEventTrackUrl = (
+  props = {},
+  eventType = null,
+) => {
```

Comment:
Should we consider making these props an object,  to improve readability when this function is invoked? (Not sure if we have coding standard for this, but I usually move to the object notation for props once there's 2+ params)

```suggestion
export const useConstructLiteSiteATIEventTrackUrl = ({
  props = {},
  eventType = null,
}) => {
```